saw-remote-api/src/SAWServer/JVMVerify.hs

```diff
@@ -0,0 +1,65 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module SAWServer.JVMVerify
+  ( jvmVerify
+  , jvmAssume
+  ) where
+
+import Prelude hiding (mod)
+import Control.Lens
+
+import qualified Language.JVM.Common as JVM
+
+import SAWScript.Crucible.JVM.Builtins
+import SAWScript.JavaExpr (JavaType(..))
+import SAWScript.Options (defaultOptions)
+import SAWScript.Value (rwCryptol)
+
+import Argo
+import CryptolServer.Data.Expression
+import SAWServer
+import SAWServer.Data.JVMType
+import SAWServer.Exceptions
+import SAWServer.JVMCrucibleSetup
+import SAWServer.OK
+import SAWServer.ProofScript
+import SAWServer.TopLevel
+import SAWServer.VerifyCommon
+
+jvmVerify :: VerifyParams JavaType -> Method SAWState OK
```

Comment:
This function and `jvmAssume` are incredibly similar. Should we worry about them getting out of sync long-term? If so, does it make sense to have them call common helpers to set up and tear down the verification task?

Reply:
Good point. I abstracted something out for both the JVM and LLVM code.

Reply:
Great, thanks!